### PR TITLE
Query update

### PR DESCRIPTION
**Pull request by HenaryKhuraijam**
## Query update
- Added a column species of type string to the animals' table.
- Updated schema.sql, data.sql, and queries.sql.

### Screenshots :
Screenshot_2023-10-28_23_53_58
Screenshot_2023-10-28_23_56_38
Screenshot_2023-10-28_23_58_06
Screenshot_2023-10-29_00_00_15
Screenshot_2023-10-29_00_04_06

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -6,4 +6,34 @@ SELECT date_of_birth from animals WHERE name IN ('Agumon','Pikachu');
...
 SELECT * from animals WHERE name != 'Gabumon';
-SELECT * from animals WHERE weight_kg >= 10.5 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT * from animals WHERE weight_kg >= 10.5 AND weight_kg <= 17.3;
+
+/* Update */
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
```
Comment (Ghiftee) [inline queries.sql]: - [ ] Amazing job creating this transaction, however, according to the requirements, please confirm the changes were made before and after rollback. e.g
```sql
   BEGIN;

   UPDATE animals
   SET species = 'unspecified'; -- make change

   SELECT species from animals; -- verify that change was made
   ROLLBACK;

   SELECT species from animals; -- verify that change was undone

File: queries.sql [hunk 2]
```diff
@@ -6,4 +6,34 @@ SELECT date_of_birth from animals WHERE name IN ('Agumon','Pikachu');
...
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name like '%mon%';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
+
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
```
Comment (Ghiftee) [inline queries.sql]: - [ ] Kindly ensure that the changes were made before and after `commit` when updating animal species. Also, before and after Rollback when deleting all animals. The purpose of verifying changes both before and after is to ensure data consistency and to check that the intended changes have been made and persisted successfully in the database.

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hello @HenaryKhuraijam 👋🏽 

Good job working on the project so far! 
To highlight 👍🏽 
✔️ Queries are successful
✔️ Screenshots are added
✔️ SQL commands are in appropriate files

However, there are some issues that you need to work on to go to the next project but you are almost there!




## Required Changes ♻️
_Check the comments under the review 👇🏽 _

- [ ] I suggest you update the highlighted section in the image below with details on how to run this specific project. This would make your README very professional and ensure that anyone who clones your project would be able to make use of it properly.
Screenshot 2023-10-28 at 20 00 53 (2)


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: queries.sql (hunk 1 above)
Comment (HenaryKhuraijam) [reply queries.sql]: Done

**Review by algerina — APPROVED** [review]
Hi @HenaryKhuraijam , 

Great job so far!  

May I highlight the following: 
- [x]  Well done implementing all the requirements👏  ✔️ 
- [x] Your code is clean ✔️



## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:

Your project is complete!  It's time to merge it :shipit:


 
 ## Optional suggestions

 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

**Pull request closed without merging**